src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String HORIZONTAL_LINES = "____________________________________________________________\n";
+    public static final String INDENTATIONS = "    ";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                INDENTATIONS + "Hello! I'm Duke\n" +
+                INDENTATIONS + "What can I do for you?\n" +
+                INDENTATIONS + HORIZONTAL_LINES);
```

Comment:
I believe a helper function like

```java
static void printlnWithIndentation(String string) {
    System.out.print(INDENTATIONS);
    System.out.println(string);
}
```

would be helpful in this `main` function to avoid code repetition, as I see a lot of `System.out.println(INDENTATIONS + "something")` in this file.